index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="styles.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <section class="container">
+      <nav>
+        <header class="toolbar">
+          <div class="logo">
+            <div class="placeholder">
+              <h3>My Logo</h3>
+            </div>
+          </div>
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the logo has a link, an `a` tag as it will be later needed to take you back to the homepage from other pages. Please implement this.